code/datums/gamemodes/distress.dm

```diff
@@ -37,6 +37,14 @@
 	for(var/obj/effect/landmark/corpsespawner/corpse AS in GLOB.corpse_landmarks_list)
 		corpse.create_mob()
 
+	for(var/i in GLOB.alive_xeno_list)
```

Comment:
You can just 'for var/xenomorph/beo AS in glob.alive_xeno_list' instead of needing to cast it later, given that the alive xeno list only contains xenos.

Reply:
Alright, done.